cmd/rocsp-tool/client_test.go

```diff
@@ -131,11 +137,13 @@ func TestLoadFromDB(t *testing.T) {
 	defer test.ResetBoulderTestDatabase(t)
 
 	for i := range 100 {
-		err = dbMap.Insert(context.Background(), &core.CertificateStatus{
-			Serial:          fmt.Sprintf("%036x", i),
-			NotAfter:        clk.Now().Add(200 * time.Hour),
-			OCSPLastUpdated: clk.Now(),
-		})
+		insertCertificateStatus(
+			t,
+			dbMap,
+			fmt.Sprintf("%036x", i),
+			clk.Now().Add(200*time.Hour),
+			clk.Now(),
+		)
```

Comment:
Optional consistency nit: I'd make this one line, like the two above.
```suggestion
		insertCertificateStatus(t, dbMap, fmt.Sprintf("%036x", i), clk.Now().Add(200*time.Hour), clk.Now())
```